README.md

```diff
@@ -1 +1 @@
-Initial Commit for the new BugBear repository.
+BB-88 Base HTML code for main page of to-do items
```

Comment:
You can just use README.md to describe your project ie. a todo list. Shouldn't have any version control related comments in here since it should be updated very few times